integration-test/src/test/java/org/ehcache/integration/statistics/TierCalculationTest.java

```diff
@@ -267,4 +276,60 @@ private void innerClear() {
     tierStatistics.clear();
     changesOf(-1, -4, -3, -2, -1);
   }
+
+  @Test
+  public void testMappingCount() {
+    assertThat(tierStatistics.getMappings()).isEqualTo(0);
+    cache.put(1, "a");
+    assertThat(tierStatistics.getMappings()).isEqualTo(1);
+  }
+
+  @Test
+  public void testMaxMappingCount() {
+    assertThat(tierStatistics.getMaxMappings()).isEqualTo(-1); // FIXME Shouldn't it be 0?
+    cache.put(1, "a");
+    cache.put(2, "b");
+    cache.remove(1);
+    assertThat(tierStatistics.getMappings()).isEqualTo(1); // FIXME: I was expecting 2
+  }
+
+  @Test
+  public void testAllocatedByteSize() {
+    assumeFalse(tierName.equals("OnHeap")); // FIXME: Not calculated for OnHeap when a size is allocated
+
+    long size = tierStatistics.getAllocatedByteSize();
+    cache.put(1, "a");
+    assertThat(tierStatistics.getAllocatedByteSize()).isGreaterThan(size); // FIXME: Why is allocated growing?
+  }
+
+  @Test
+  public void testOccupiedByteSize() {
+    assertThat(tierStatistics.getOccupiedByteSize()).isEqualTo(0);
+    cache.put(1, "a");
+    assertThat(tierStatistics.getOccupiedByteSize()).isGreaterThan(0);
+  }
+
+  @Test
+  public void testEviction() {
+    String payload = "aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa";
+
+    // Wait until we reach the maximum that we can fit in to make sure that are indeed evictions calculated
+    int i = 0;
+    long evictions;
+    do {
+      cache.put(i++, payload);
+      evictions = tierStatistics.getEvictions();
+    }
+    while(evictions == 0 && i < 100_000); // The 100 000 threshold is to prevent an infinite loop in case of a bug
+
+    assertThat(evictions).isGreaterThan(0);
+  }
+
+  @Test
```

Comment:
Does this test pass?  At the moment I am not seeing any ExpirationCount values > 0 .  I am checking via the TMS REST api.